src/lib/components/batch-operations/results.svelte

```diff
@@ -1,9 +1,9 @@
 <script lang="ts">
-  import { translate } from '$lib/i18n/translate';
+  import { createTranslate } from '$lib/i18n/translate';
   import type { BatchOperation } from '$lib/types/batch';
 
   export let operation: BatchOperation;
-
+  const t = createTranslate('batch');
```

Comment:
Alternatively, another possibility would be something like:

```ts
const { results, operationsProgess, operationsSucceeded, operationsFailed } = createTranslate('batch');
```

Reply:
Yeah that would be nice, let me look into this.

Reply:
Team opted not to do this as it will end up creating polluting the `<script />` tag in our svelte components with lots of on-off variable names. Also would likely require a refactor of all of our i18n keys to be `camelCase` instead of `kebab-case`.